### PR TITLE
fix: adapt color palette to new themes

### PR DESCRIPTION
**Pull request by sosa-vaadin**
## Description

This PR updates the color palette for Lumo Light Theme to the the new Lumo theme for charts introduced in https://github.com/vaadin/web-components/pull/3006.

### CODE REVIEW

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 1]
```diff
@@ -32,39 +32,38 @@
 public class LumoLightTheme extends Theme {
 
     protected static final int BORDER_RADIUS = 5;
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:34]: I assume this is pixels. The `--lumo-border-radius-m` property is by default `0.25rem` which equals `4px` with browser default styles. Though, I’m not sure where this property is used, if it makes sense to align it.

```suggestion
    protected static final int BORDER_RADIUS = 4;
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 2]
```diff
@@ -32,39 +32,38 @@
...
+    protected final static SolidColor TOOLTIP_TEXT_COLOR = new SolidColor(24,
+            39, 57, 0.94);
     protected final static SolidColor TOOLTIP_BACKGROUND_COLOR = new SolidColor(
             255, 255, 255);
-    protected final static SolidColor SUBTITLE_COLOR = new SolidColor(128, 128,
-            128);
+    protected final static SolidColor SUBTITLE_COLOR = new SolidColor(27, 43,
+            65, 0.72d);
     protected final static SolidColor LINE_COLOR = new SolidColor(223, 223,
             223);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:60]: `--lumo-contrast-20pct`

```suggestion
    protected final static SolidColor LINE_COLOR = new SolidColor(28, 55, 90, 0.16);
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 3]
```diff
@@ -32,39 +32,38 @@
...
     protected final static SolidColor TEXT_COLOR = new SolidColor(128, 128,
             128);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:62]: `--lumo-secondary-text-color`

```suggestion
    protected final static SolidColor TEXT_COLOR = new SolidColor(27, 43, 65, 0.69);
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 4]
```diff
@@ -32,39 +32,38 @@
...
-            250);
-    protected final static SolidColor LABEL_COLOR = new SolidColor(128, 128,
-            128);
+    protected final static SolidColor GRID_COLOR = new SolidColor(28, 55, 90,
+            0.16d);
+    protected final static SolidColor LABEL_COLOR = new SolidColor(24, 39, 57,
+            0.94d);
 
     protected final static SolidColor BGCOLOR = new SolidColor(255, 255, 255);
     protected final static SolidColor BGCOLOR_LIGHT_GRAY = new SolidColor(0, 0,
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:69]: `--lumo-contrast-5pct`

Couldn't extend the suggestion on the second line, please fix locally.

```suggestion
    protected final static SolidColor BGCOLOR_LIGHT_GRAY = new SolidColor(25, 59, 103, 0.05);
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 5]
```diff
@@ -73,8 +72,8 @@
             255, 255, 0.0);
 
     protected final static String DEFAULT_FONT_FAMILIES = "var(--lumo-font-family)";
-    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(128, 128,
-            128);
+    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(255, 255,
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:75]: Is the legend text on a dark background? This color is now a semi-transparent white.

Comment (sosa-vaadin) [reply vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:75]: I believe it is on a full black background, the one in line 68:
```java
protected final static SolidColor BGCOLOR = new SolidColor(255, 255, 255);
```

Comment (jouni) [reply vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:75]: 255,255,255 is white 😄

Comment (sosa-vaadin) [reply vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:75]: Then maybe we should keep the text light gray or black.

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 6]
```diff
@@ -31,50 +31,49 @@
...
-    protected final static SolidColor TEXT_COLOR = new SolidColor(128, 128,
-            128);
-    protected final static SolidColor GRID_COLOR = new SolidColor(250, 250,
-            250);
-    protected final static SolidColor LABEL_COLOR = new SolidColor(128, 128,
-            128);
+    protected final static SolidColor SUBTITLE_COLOR = new SolidColor(27, 43,
+            65, 0.72d);
+    protected final static SolidColor LINE_COLOR = new SolidColor(28, 55, 90,
+            0.16);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:60]: Some of these have `d`, while others don't (`0.72d` vs `0.16`). I suppose they should be unified?

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 7]
```diff
@@ -31,50 +31,49 @@
...
+    protected final static SolidColor TEXT_COLOR = new SolidColor(27, 43, 65,
+            0.69);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:62]:
```suggestion
    protected final static SolidColor TEXT_COLOR = new SolidColor(27, 43,
              65, 0.72d);
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 8]
```diff
@@ -31,50 +31,49 @@
...
+    protected final static SolidColor GRID_COLOR = new SolidColor(28, 55, 90,
+            0.16d);
+    protected final static SolidColor LABEL_COLOR = new SolidColor(24, 39, 57,
+            0.94d);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:66]:
```suggestion
    protected final static SolidColor LABEL_COLOR = new SolidColor(27, 43,
              65, 0.72d);
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 9]
```diff
@@ -31,50 +31,49 @@
...
     protected final static SolidColor BGCOLOR = new SolidColor(255, 255, 255);
-    protected final static SolidColor BGCOLOR_LIGHT_GRAY = new SolidColor(0, 0,
-            0, 0.02);
+    protected final static SolidColor BGCOLOR_LIGHT_GRAY = new SolidColor(25,
+            59, 103, 0.05);
     protected final static SolidColor TRANSPARENT_COLOR = new SolidColor(255,
             255, 255, 0.0);
 
     protected final static String DEFAULT_FONT_FAMILIES = "var(--lumo-font-family)";
     protected static final Color LEGEND_TEXT_COLOR = new SolidColor(128, 128,
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:75]:
```suggestion
    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(27, 43,
              65, 0.72d)
```

**Review by jouni — COMMENTED** [review]
Now when I looked at the previous colors, I realized that most of the text colors were the same light gray, so I changed those to use the "secondary text color".

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 10]
```diff
@@ -72,7 +72,8 @@
             255, 255, 0.0);
 
     protected final static String DEFAULT_FONT_FAMILIES = "var(--lumo-font-family)";
-    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(128, 128,
+    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(27, 43,
+              65, 0.72d)
             128, 0.94);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java]: This was a leftover from my suggestion, as GitHub didn't allow me to extend a suggestion over a deleted line.

```suggestion
```

File: vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java [hunk 11]
```diff
@@ -31,50 +31,49 @@
...
+    protected final static SolidColor BGCOLOR_LIGHT_GRAY = new SolidColor(25,
+            59, 103, 0.05);
     protected final static SolidColor TRANSPARENT_COLOR = new SolidColor(255,
             255, 255, 0.0);
 
     protected final static String DEFAULT_FONT_FAMILIES = "var(--lumo-font-family)";
-    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(128, 128,
-            128);
+    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(27, 43, 128,
+            0.94);
```
Comment (jouni) [inline vaadin-charts-flow-parent/vaadin-charts-flow/src/main/java/com/vaadin/flow/component/charts/themes/LumoLightTheme.java:76]: One more time 😅  Sorry for being a little sloppy with the review.

```suggestion
    protected static final Color LEGEND_TEXT_COLOR = new SolidColor(27, 43, 65,
            0.72);
```

**Review by jouni — APPROVED** [review]

**Pull request merged**